pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
```

Comment:
This comment should really be tied to our initial pull request in #1 but we'll take a look and see how to implement it